Makefile

```diff
@@ -0,0 +1,45 @@
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
+# Targets
+
+.PHONY: all
+all: build
```

Comment:
I don't see the `build` target. Is `build` target left by mistake or is it an intrinsic target in make? I'm not aware of it being the latter 🤔

Reply:
Whoops! Yep meant to remove that one! Thank you!